linux-armv7-lts: cmake error when building fmtlib with conan

Hi,

I'm using latest version of linux-armv7-lts to build fmt library from sources and it seems that there is a problem with cmake. When I am doing the same thing (conan install .. --build=fmt) on x86_64 host then it works fine (output is the same, but without an error). Moreover, a few months ago I have used build tag 20220409-7e72803 of this image and it worked fine, but now fmt lib requires more recent version of conan and I cannot use it anymore. I'm not sure if this is an issue in image or I'm missing something?

```
(mydev) 20:27:31 ✗ user1:user1-VirtualBox: ~/projects/docker_build/test_fmtlib/build $
~/projects/docker_build/dockcross-linux-armv7-lts --args "-v /home/user1/.conan:/home/user1/.conan -v /home/user1/projects/docker_build/test_fmtlib:/work/test_fmtlib" bash -c "cd test_fmtlib/build; conan install .. -pr=armv7_dd --build=fmt"
WARN: Migration: Updating settings.yml
WARN: ****************************************
WARN: settings.yml is locally modified, can't be updated
WARN: The new settings.yml has been stored in: /home/user1/.conan/settings.yml.new
WARN: ****************************************
Configuration:
[settings]
arch=armv7
arch_build=armv7
build_type=Release
compiler=gcc
compiler.libcxx=libstdc++11
compiler.version=8
os=Linux
os_build=Linux
[options]
[build_requires]
[env]

fmt/8.1.1: Forced build from source
conanfile.txt: Installing package
Requirements
    fmt/8.1.1 from 'conancenter' - Cache
Packages
    fmt/8.1.1:947982ce50bcd84e91dc2842b484e5adcbf9a0b0 - Build

Installing (downloading, building) binaries...
fmt/8.1.1: Copying sources to build folder
fmt/8.1.1: Building your package in /home/user1/.conan/data/fmt/8.1.1/_/_/build/947982ce50bcd84e91dc2842b484e5adcbf9a0b0
fmt/8.1.1: Generator txt created conanbuildinfo.txt
fmt/8.1.1: Calling generate()
fmt/8.1.1: WARN: Using the new toolchains and generators without specifying a build profile (e.g: -pr:b=default) is discouraged and might cause failures and unexpected behavior
fmt/8.1.1: Aggregating env generators
fmt/8.1.1: Calling build()
fmt/8.1.1: CMake command: cmake -G "Unix Makefiles" -DCMAKE_TOOLCHAIN_FILE="/home/user1/.conan/data/fmt/8.1.1/_/_/build/947982ce50bcd84e91dc2842b484e5adcbf9a0b0/build/generators/conan_toolchain.cmake" -DCMAKE_INSTALL_PREFIX="/home/user1/.conan/data/fmt/8.1.1/_/_/package/947982ce50bcd84e91dc2842b484e5adcbf9a0b0" -DFMT_DOC="OFF" -DFMT_TEST="OFF" -DFMT_INSTALL="ON" -DFMT_LIB_DIR="lib" -DFMT_OS="ON" -DCMAKE_POLICY_DEFAULT_CMP0091="NEW" -DCMAKE_BUILD_TYPE="Release" "/home/user1/.conan/data/fmt/8.1.1/_/_/build/947982ce50bcd84e91dc2842b484e5adcbf9a0b0/src"
-- CMake version: 3.23.1
-- Using Conan toolchain: /home/user1/.conan/data/fmt/8.1.1/_/_/build/947982ce50bcd84e91dc2842b484e5adcbf9a0b0/build/generators/conan_toolchain.cmake
-- Conan toolchain: Setting CMAKE_POSITION_INDEPENDENT_CODE=ON (options.fPIC)
-- Conan toolchain: Setting BUILD_SHARED_LIBS = OFF
-- The CXX compiler identification is GNU 8.5.0
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/xcc/armv7-unknown-linux-gnueabi/bin/armv7-unknown-linux-gnueabi-g++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Module support is disabled.
-- Version: 8.1.1
-- Build type: Release
-- CXX_STANDARD: 11
-- Performing Test has_std_11_flag
-- Performing Test has_std_11_flag - Success
-- Performing Test has_std_0x_flag
-- Performing Test has_std_0x_flag - Success
-- Performing Test SUPPORTS_USER_DEFINED_LITERALS
-- Performing Test SUPPORTS_USER_DEFINED_LITERALS - Success
-- Performing Test FMT_HAS_VARIANT
-- Performing Test FMT_HAS_VARIANT - Success
-- Required features: cxx_variadic_templates
-- Looking for strtod_l
-- Looking for strtod_l - found
-- Configuring done
-- Generating done
CMake Warning:
  Manually-specified variables were not used by the project:

    CMAKE_POLICY_DEFAULT_CMP0091


-- Build files have been written to: /home/user1/.conan/data/fmt/8.1.1/_/_/build/947982ce50bcd84e91dc2842b484e5adcbf9a0b0/build/Release
fmt/8.1.1: CMake command: cmake --build "/home/user1/.conan/data/fmt/8.1.1/_/_/build/947982ce50bcd84e91dc2842b484e5adcbf9a0b0/build/Release" '--' '-j4'
[ 33%] Building CXX object CMakeFiles/fmt.dir/src/format.cc.o
[ 66%] Building CXX object CMakeFiles/fmt.dir/src/os.cc.o
[100%] Linking CXX static library libfmt.a
[100%] Built target fmt
fmt/8.1.1: Package '947982ce50bcd84e91dc2842b484e5adcbf9a0b0' built
fmt/8.1.1: Build folder /home/user1/.conan/data/fmt/8.1.1/_/_/build/947982ce50bcd84e91dc2842b484e5adcbf9a0b0/build/Release
fmt/8.1.1: Generated conaninfo.txt
fmt/8.1.1: Generated conanbuildinfo.txt
fmt/8.1.1: Generating the package
fmt/8.1.1: Package folder /home/user1/.conan/data/fmt/8.1.1/_/_/package/947982ce50bcd84e91dc2842b484e5adcbf9a0b0
fmt/8.1.1: Calling package()
fmt/8.1.1: Copied 1 '.rst' file: LICENSE.rst
fmt/8.1.1: CMake command: cmake --install "/home/user1/.conan/data/fmt/8.1.1/_/_/build/947982ce50bcd84e91dc2842b484e5adcbf9a0b0/build/Release" --prefix "/home/user1/.conan/data/fmt/8.1.1/_/_/package/947982ce50bcd84e91dc2842b484e5adcbf9a0b0"
CMake Warning:
  Ignoring extra path from command line:

   "/home/user1/.conan/data/fmt/8.1.1/_/_/package/947982ce50bcd84e91dc2842b484e5adcbf9a0b0"


CMake Error: Unknown argument --prefix
CMake Error: Run 'cmake --help' for all supported options.
ERROR: fmt/8.1.1: Error in package() method, line 93
        cmake.install()
        ConanException: Error 1 while executing cmake --install "/home/user1/.conan/data/fmt/8.1.1/_/_/build/947982ce50bcd84e91dc2842b484e5adcbf9a0b0/build/Release" --prefix "/home/user1/.conan/data/fmt/8.1.1/_/_/package/947982ce50bcd84e91dc2842b484e5adcbf9a0b0"
```

--- a/imagefiles/cmake.sh
+++ b/imagefiles/cmake.sh
@@ -10,10 +10,11 @@
 #   -E                           = CMake command mode.
 #   --build <dir>                = Build a CMake-generated project binary tree.
 #   --find-package               = Run in pkg-config like mode.
+#   --install                    = install an already-generated project
 #
 case $1 in
 
-  -E|--build|--find-package)
+  -E|--build|--find-package|--install)
       exec /usr/bin/cmake "$@"
       ;;
 

